src/include/gurt/errno.h

```diff
@@ -170,27 +170,77 @@
 	/** Checksum error */						\
 	ACTION(DER_CSUM,		(DER_ERR_DAOS_BASE + 21))
 
+#if defined(D_ERRNO_V2)
+/** Defines the gurt error codes */
+#define D_FOREACH_ERR_RANGE(ACTION)	\
+	ACTION(GURT,	1000)
+#else /* !D_ERRNO_V2 */
+/** Defines the gurt error codes */
 #define D_FOREACH_ERR_RANGE(ACTION)	\
 	ACTION(GURT,	1000)		\
 	ACTION(DAOS,	2000)
+#endif /* D_ERRNO_V2 */
+
+#define D_DEFINE_ERRNO(name, value) name = value,
+
+#define D_DEFINE_RANGE_ERRNO(name, base)			\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon